MIT.md

```diff
@@ -0,0 +1,7 @@
+## Copyright 2021, [VICTOR ADERIBIGBE]
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this Leaderboard setup and associated documentation files, to deal in the Leaderboard without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Leaderbaord, and to permit persons to whom the Leaderboard is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Leaderboard.
+
+THE Leaderboard IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [APP TYPE] OR THE USE OR OTHER DEALINGS IN THE Leaderboard.
```

Comment:
@Bria222 Nice work partner, Kindly edit this md license to meet the present situation.